src/Workshop/DemoBundle/Component/Print/LegendHandler.php

```diff
@@ -0,0 +1,28 @@
+<?php
+    namespace Workshop\DemoBundle\Component\Print;
+
+use Mapbender\PrintBundle\Component\TemplateRegion;
+
+class LegendHandler extends \Mapbender\PrintBundle\Component\LegendHandler
+{
+
+    public function __construct(
+        #[Autowire(service: 'mapbender.imageexport.image_transport.service')] $imageTransport,
+        #[Autowire('mapbender.print.resource_dir')] string                                   $resourceDir,
+        #[Autowire('mapbender.print.temp_dir')] ?string                                      $tmpDir,
+        #[Autowire('mapbender.print.canvas_legend.class')] ?string                           $canvasLegendClass
+    )
+    {
+        parent::__construct($imageTransport, $resourceDir, $tmpDir, $canvasLegendClass);
+        $this->dynamicColumnSizes = true;
+        $this->dynamicColumnWidths = true;
```

Comment:
`dynamicColumnWidths` gibt es nicht, `dynamicColumnSizes` ist richtig.

Reply:
ah. strange why I used it. I will fix it.

I saw $maxColumnWidthMm - I would add it. right?

Reply:
You can change that, yes. Default is 100 (10cm), so on A4 portrait it fits two columns on a page (respecting the margins)